core/modules/contact/contact.admin.inc

```diff
@@ -24,33 +24,40 @@ function contact_settings_form($form, &$form_state) {
   );
   $form['fields']['copy'] = array(
     '#type' => 'checkbox',
-    '#title' => t("Show the 'Send yourself a copy' checkbox"),
-    '#description' => t("Allow users to send themselves a copy on site-wide and personal contact forms."),
+    '#title' => t("Allow users to send themselves a copy of the submitted data."),
+    '#description' => t("Exposes a 'Send yourself a copy' checkbox on site-wide and personal contact forms."),
     '#default_value' => contact_show_personal_copy_checkbox()
   );
 
   $form['threshold'] = array(
     '#type' => 'fieldset',
+    '#title' => t("Send limit"),
     '#collapsible' => TRUE,
     '#collapsed' => TRUE,
-    '#title' => t("Send limit"),
-    '#description' => t("Configure thresholds for the contact forms to minimize malicious users sending multiple emails. Control the number of emails that can be sent in a window of time by anyone with the same IP address."),
   );
   $form['threshold']['wrapper'] = array(
     '#type' => 'container',
     '#attributes' => array('class' => array('container-inline')),
   );
   $form['threshold']['wrapper']['contact_threshold_limit'] = array(
     '#type' => 'select',
-    '#title' => t('Limit to'),
+    '#title' => t('Email send limit'),
+    '#title_display' => 'invisible',
     '#options' => backdrop_map_assoc(array(1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 20, 30, 40, 50, 75, 100, 125, 150, 200, 250, 500)),
     '#default_value' => $config->get('contact_threshold_limit'),
+    '#prefix' => t('Limit to'),
   );
   $form['threshold']['wrapper']['contact_threshold_window'] = array(
     '#type' => 'select',
-    '#title' => t('messages per'),
+    '#title' => t('Email send time window'),
+    '#title_display' => 'invisible',
     '#options' => array(0 => t('None (disabled)')) + backdrop_map_assoc(array(60, 180, 300, 600, 900, 1800, 2700, 3600, 10800, 21600, 32400, 43200, 86400), 'format_interval'),
     '#default_value' => $config->get('contact_threshold_window'),
+    '#prefix' => t('messages per'),
+  );
+  $form['threshold']['help'] = array(
+    '#type' => 'item',
```

Comment:
What about using the `#type => 'help'` ? https://api.backdropcms.org/form_api#help

Reply:
Sounds like something else we should fix though!